corehq/apps/auditcare/utils/migration.py

```diff
@@ -29,6 +29,8 @@ def generate_batches(self, worker_count, batch_by):
             start_datetime = self.get_next_batch_start()
             if not start_datetime:
                 # for the first call
+                if AuditcareMigrationMeta.objects.count() != 0:
+                    raise Exception("Unable to get start time. Exiting.")
```

Comment:
If this needs to be caught then the best practice is to use a more specific class than `Exception` since it's impossible to catch this without possibly catching other exceptions that you do not want.

It's trivial to create a custom exception to raise here and then catch elsewhere.
```py
class MissingStartTimeError(Exception):
    pass
```